src/components/atoms/NavLink.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <a
+    :href="href"
+    class="nav-link active text-right"
```

Comment:
I think to avoid positioning properties in atoms (Strictly in the component).  

So, should be passed as parent component props for variability.

https://rscss.io/layouts.html

Reply:
親で位置を指定するように変更しました。